Gododir/gen.go

```diff
@@ -506,15 +507,23 @@ func vet(service *service) {
 }
 
 func storageVersion(c *do.Context) {
-	versionFile := "storage/version.go"
+	version("storage")
```

Comment:
I guess it makes sense to have this change for now. However, once everything has been submoduled out, managementVersion and storageVersion should no longer be present.